membership_initial_fee/models/account_invoice.py

```diff
@@ -32,11 +32,11 @@ def _prepare_initial_fee_vals(self):
         return inv_line._convert_to_write(inv_line._cache)
 
     @api.multi
-    def initial_fee_create_check(self, product):
```

Comment:
Mmm, I get your point here, but you should keep compatibility with other possible modules using/extending this method.

Something like:
```python
def initial_fee_create_check(self, product=False):
    # TODO: remove product parameter in v12
    product = product or self
    if not product or not product.membership or product.initial_fee == 'none':
        return False
```

Reply:
Yes, It makes total sense. Thanks for the feedback. I'll push the changes according to your suggestion.